tests/host_tools/network.py

```diff
@@ -295,6 +297,12 @@ def get_guest_net_if_name(ssh_connection, guest_ip):
     return if_name if if_name != "" else None
 
 
+def random_str(k):
+    """Create a random string of length `k`."""
+    symbols = string.ascii_lowercase + string.digits
+    return "".join(random.choices(symbols, k=k))
+
+
 class Tap:
```

Comment:
Just for extra measure, could we add a destructor here that will remove the tap device (the one outside the namespace) when the object gets dropped?

Reply:
But when it's moved it disappears from the default namespace no? Actually the docs of the class say it will get removed when it goes out of scope but I don't see anything related to that.

Reply:
Does it? I'm not saying it doesn't, just lack the knowledge. If it does, that's fine by me.

Reply:
I guess the netns gets destroyed when the test ends, and anything dangling will get removed after Docker exits.